docs/sources/observability-as-code/get-started.md

```diff
@@ -17,44 +17,46 @@ weight: 100
 
 # Get started with Observability as Code
 
-Simply put, with Observability as Code, you can manage Grafana resources.
-You can write code that describes what you want the dashboard to do, rather than manipulate it via the UI.
+Grafana provides a suite of tools for **Observability as Code** to help you manage your Grafana resources programmatically and at scale. This approach allows you to define dashboards, data sources, and other configurations in code, enabling version control, automated testing, and reliable deployments through CI/CD pipelines.
 
-Observability as Code lets you manage dashboards, resources, and configurations programmatically, leveraging powerful tools for automation and standardization.
+Historically, managing Grafana as code involved various community and Grafana Labs tools, but lacked a single, cohesive story. With Grafana 12, we're introducing foundational improvements, including **new versioned APIs** and official tooling, to provide a clearer path forward.
```

Comment:
This sentence is so much better and adds context -- the why someone should care. Do we need to bold ver versioned APIs? Usually, we only bold UI element names. 
```suggestion
Historically, managing Grafana as code involved various community and Grafana Labs tools, but lacked a single, cohesive story. Grafana 12 introduces foundational improvements, including new versioned APIs and official tooling, to provide a clearer path forward.
```